Add awesome-coronavirus

soroushchehresa opened the pull request.

jnv commented:
Thanks for the PR Soroush, love the logo! I have moved the project to the `awesome-*` section and modified the description a bit; from my understanding 2019-nCoV/SARS-CoV-2 and COVID-19 are two different things, so I wanted the description to reflect that more clearly.

soroushchehresa commented:
> Thanks for the PR Soroush, love the logo! I have moved the project to the `awesome-*` section and modified the description a bit; from my understanding 2019-nCoV/SARS-CoV-2 and COVID-19 are two different things, so I wanted the description to reflect that more clearly.

@jnv Thanks.
